packages/react-instantsearch-core/src/connectors/__tests__/connectInfiniteHits.js

```diff
@@ -498,17 +498,6 @@ describe('connectInfiniteHits', () => {
       expect(instance.refine).toHaveBeenLastCalledWith(event, 1);
     });
 
-    it('adds 1 to page when calling refine', () => {
```

Comment:
`refine` is not a public method, so we don't need to test it. And we don't need this behavior either.